src/App.js

```diff
@@ -56,6 +56,7 @@ export default function App() {
     setPadIsTriggered(key, true)
     const padName = getPadNameByKey(key)
     setSelectedPad(padName)
+    //.. setSelectedSample(samplePlayer -> key)
```

Comment:
Is this comment still needed? Maybe you forgot to delete it.

Reply:
Yes, thanks i forgot it.